pyproject.toml

```diff
@@ -20,6 +20,8 @@ classifiers = [
   "Programming Language :: Python :: 3.9",
   "Programming Language :: Python :: 3.10",
   "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13",
```

Comment:
Please also add new versions to the `cd.yml` matrix

Reply:
Done.